### DISCUSSION TITLE
YouTube demo is broken on Playground

### DISCUSSION THREAD

**Discussion by thinkmorebetter** [deepgram/community#849] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - STT Pre-recorded

### Details

The Playground at https://playground.deepgram.com/ used to work for YouTube links, but now there's a 500 error on form submission. We're also seeing errors internally when submitting a YouTube stream.

Example video URL: https://www.youtube.com/watch?v=75NQK-Sm1YY

<img width="1726" alt="2024-07-10_12-48-25" src="https://github.com/deepgram/community/assets/963801/aaa002d2-ba53-4bae-bf7e-ff923b6928f6">

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@thinkmorebetter  thanks for reporting this, I could reproduce and see that there is 500 error coming from the service that processes the Youtube URLs.  I'm notifying one of our engineering teams now.

**Reply by jpvajda** [reply, marked as answer]
@thinkmorebetter  We currently are experiencing issues with a library we use to interface with YouTube and will be disabling this feature in our API Playground until we can resolve it.

Sorry for the inconvenience here.

**Reply by jpvajda** [reply]
Just an update: We were able to quickly resolve the issue, so YouTube Transcription in the Playground should be working again @thinkmorebetter

**Reply by thinkmorebetter** [reply]
@jpvajda would you mind sharing how you were able to fix it? We have a broken YouTube integration as well as many others.

**Reply by jpvajda** [reply]
Sure. @thinkmorebetter  it looks like we had to switch from using   `ytdl-core` to `@distube/ytdl-core` version 4.13.15
